src/nl/hannahsten/texifyidea/editor/typedhandlers/LatexEnterInEnumerationHandler.kt

```diff
@@ -35,7 +35,7 @@ class LatexEnterInEnumerationHandler : EnterHandlerDelegate {
         }
 
         // Don't insert \item when the enter was triggered by the word wrap
-        if (DataManager.getInstance().loadFromDataContext(context, AutoHardWrapHandler.AUTO_WRAP_LINE_IN_PROGRESS_KEY) == true) {
+        if (DataManager.getInstance().loadFromDataContext(context, LineWrappingUtil.WRAP_LONG_LINE_DURING_FORMATTING_IN_PROGRESS_KEY) == true) {
```

Comment:
This indeed seems to be the right key, except that we need to check for _both_ as they serve a different purpose: one is for when the line is wrapping while you are typing, and the other is for when the line is wrapped if you trigger a reformat.
It should be possible to write a test, I think.

However, I see that the key is marked as internal api, so we should ask the IntelliJ team for guidance on what we should use instead of this, by posting a question at https://intellij-support.jetbrains.com/hc/en-us/community/topics/200366979-IntelliJ-IDEA-Open-API-and-Plugin-Development
However, it sounds awfully similar to a discussion we had with the IntelliJ team regarding #2838, with the result of the LatexLineWrapper class. Maybe we should be reusing that here as well (not sure exactly how it would work, maybe you want to figure that out?).

Reply:
I asked about it, and in the meantime i replaced the old way

Reply:
Thanks, let's wait for an answer there.
For reference: https://intellij-support.jetbrains.com/hc/en-us/community/posts/12261976643730-How-to-detect-EnterHandlerDelegate-line-wrap-during-format